### ISSUE TITLE
Add more visual structure

### ISSUE THREAD

**Issue by snomos** [UAlbertaALTLab/morphodict#1014]
![image](https://user-images.githubusercontent.com/1255285/140016281-138ba8ce-c432-47f4-8661-113034b4efde.jpeg)

See screen shot. I would suggest to separate the three blocks more clearly: standard paradigm, diminutives and possessives. And also make columns and rows more visually salient.

As it is now, the paradigm table is very much (visually) an unstructured blob of text - there is very little visual support for the actual structure.

**Comment by aarppe** [comment]
Yes, this is true, and have an all too long-standing issue concerning this #568, based on user feedback in #567 (with some sketches for improving this that we have to work through).
